my-first-app/src/Components/PostCard/PostCard.tsx

```diff
@@ -0,0 +1,93 @@
+import Like from '../../assets/Icon-Thumbs-Up.svg?react'
+import DisLike from '../../assets/Icon-Thumbs-Down.svg?react'
+import Save from '../../assets/Icon-Bookmark.svg?react'
+
+import style from './PostCard.module.scss'
+
+interface IProps {
+  size: 'sizeL' | 'sizeM' | 'sizeS',
+  date: string,
+  title: string,
+  description: string,
+  image?: string,
+  onClick: () => void,
+}
+const PostCard = ({ size, date, title, description, image, onClick }: IProps) => {
+
+  switch (size) {
+    case 'sizeL':
```

Comment:
inside in PostCard folder for each rendering case create seperated components.example: sizeLcomponent,SizeMcomponent and use them here